.cilibs/compile_code.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+GATE_VERSION=release-1.20.x
+BUILD_OS=linux
+BUILD_ARCH=amd64
+
+
+while getopts "o:a:g:" opt; do
+  case ${opt} in
+    o) #Build OS
+      BUILD_OS=${OPTARG}
+      ;;
+    a) #Build arch
+      BUILD_ARCH=${OPTARG}
+      ;;
+    g) #Gate version
+      GATE_VERSION=${OPTARG}
+      ;;
+  esac
+done
+
+echo "Compile code"
+if [ -z "$TRAVIS_BRANCH" ]
+then
+    export RELEASE=$(echo $CIRCLE_TAG | sed 's/^v[0-9]\+\.[0-9]\+\.[0-9]\+-\?//')
+else
+    export RELEASE=$CIRCLE_BRANCH
+fi
```

Comment:
This is used for release pipeline which is executed against tag, not branch. How will this work in Travis?

Reply:
Documantation says it should work:
`TRAVIS_BRANCH: for builds triggered by a tag, this is the same as the name of the tag (TRAVIS_TAG)`